src/app/command-processor.test.js

```diff
@@ -22,6 +22,8 @@ const commandProcessor = require('./command-processor');
 
 
 describe('command-line parsing', () => {
+	const sandbox = sinon.createSandbox();
```

Comment:
My understanding is that `sinon` itself is a sandbox so typically you don't need to create these, and if you do create them, typically you use `sandbox.restore()` to undo stubs/mocks all at once. The stubs you create using this sandbox look to just be on temporary objects so you don't have to restore them, which makes me very curious why you create this `sandbox` instead of using `sinon` directly?

Reply:
> if you do create them, typically you use sandbox.restore() to undo stubs/mocks all at once.

ah, yeah. forgot that bit. nice catch :+1:

> why you create this sandbox instead of using sinon directly?

eh, just out of habit - i find it's easier to have one explicit pattern then remember special cases - not a strong preference

Reply:
The mix of using `sinon` for some of the stubs and `sandbox` for others is what confuses me mostly. If it is just an incremental transition to using sandboxes more, I'm cool with that, both just seems weird without the context.

Reply:
lol just noticed that. i'll clean it up :+1:

Reply:
Trap for the unwary: some of our JS projects use an older version of sinon where `sinon` isn't a sandbox. If you run into this, please either upgrade it or log a story so we know to clean it up.